src/components/BottomSheet.js

```diff
@@ -3,10 +3,9 @@ import { useState } from 'react';
 
 import './BottomSheet.css';
 
-function BottomSheet() {
+function BottomSheet(props) {
```

Comment:
```suggestion
function BottomSheet({ data }) {
```
취향차이긴 하지만 이게 조금 더 일반적으로 사용되는 방식인거 같습니다.